waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates.hs

```diff
@@ -159,19 +164,20 @@ openSaasStarterTemplate =
 styleText :: String -> String
 styleText = id
 
-simpleGhRepoTemplate :: (String, Path' Rel' Dir') -> (String, String) -> StartingInstructionsBuilder -> StarterTemplate
-simpleGhRepoTemplate (repoName, tmplPathInRepo) (tmplDisplayName, tmplDescription) buildStartingInstructions =
-  GhRepoStarterTemplate
+simpleGhReleaseAssetTemplate :: (String, GhRepo.GithubReleaseAssetName) -> (String, String) -> StartingInstructionsBuilder -> StarterTemplate
+simpleGhReleaseAssetTemplate (repoName, assetName) (tmplDisplayName, tmplDescription) buildStartingInstructions =
+  GhRepoReleaseAssetTemplate
     ( GhRepo.GithubRepoRef
         { GhRepo._repoOwner = waspGhOrgName,
           GhRepo._repoName = repoName,
           GhRepo._repoReferenceName = waspVersionTemplateGitTag
         }
     )
+    assetName
     ( DirBasedTemplateMetadata
         { _name = tmplDisplayName,
           _description = tmplDescription,
-          _path = tmplPathInRepo,
+          _path = [reldir|.|],
```

Comment:
Given this function is called _`simple`_`GhReleaseAssetTemplate`, I'm assuming the tarball has the files in the top level. This is easy to adapt if that changes in the future.

Reply:
https://github.com/wasp-lang/open-saas/pull/528